### PR TITLE
c36  character personality translation

### PR DESCRIPTION
**Pull request by ScrewThisNoise**

### CODE REVIEW

**Review by ManlyMarco — CHANGES_REQUESTED** [review]
Remove the triple extra newlines at the ends of files and it's good.

**Comment by ScrewThisNoise** [comment]
@ManlyMarco Fixed triple newlines

**Review by ManlyMarco — APPROVED** [review]

File: translation/en/RedirectedResources/assets/abdata/communication/info_20/optiondisplayitems_36/translation.txt [hunk 1]
```diff
@@ -1,74 +1,36 @@
...
-高いのが=Tall ones
-ぽっちゃりしているのが=Chubby ones
-痩せているのが=Thin ones
-うーん、そうかも？=Yeah、probably?
-そんな事ないよ=That's not true at all
-バッチリ！　１０年先まで！=Exactly! I plan 10 years ahead!
-実際何も考えてないです……=Actually I don't think much about it...
-それはまぁ……してみたいかな=Well... I think I'd like to try
-いや、それはないかな=Nah、rather not
-
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/communication/info_20/optiondisplayitems_36/translation.txt:37]: This part of new dump got removed in the process, it should be added back

File: translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_36/translation.txt [hunk 2]
```diff
@@ -1,123 +1,51 @@
...
 本当？　嬉しいよ=Really? I'm glad.
 いいよ=Alright.
 優しくしてもらえると=If you could please be gentle...
-出来ています=I'm ready.
-了解です=Understood.
+出来ています=It's ready.
+了解です=Roger that!
 わかりました=I understand.
 ありがとうございます=Thank you.
-
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_36/translation.txt:52]: This got removed too

**Review by ManlyMarco — CHANGES_REQUESTED** [review]

File: translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_36/translation.txt (hunk 2 above)
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/communication/info_26/optiondisplayitems_36/translation.txt:52]: fixed

File: translation/en/RedirectedResources/assets/abdata/communication/info_20/optiondisplayitems_36/translation.txt (hunk 1 above)
Comment (ScrewThisNoise) [reply translation/en/RedirectedResources/assets/abdata/communication/info_20/optiondisplayitems_36/translation.txt:37]: fixed

**Review by ManlyMarco — APPROVED** [review]

**Pull request closed without merging**